wiki/en/en-Installation-for-Linux.md

```diff
@@ -29,18 +29,12 @@ tar -xvf latest.tar.gz
 
 First, update your package list (e.g. on Debian based distributions with `sudo apt-get update`).
 
-On **Ubuntu-based** distributions 18.04+, and on Debian 9 or 10:
+On **Ubuntu-based** distributions 18.04+, **Debian** 9 or later,  Raspberry Pi Raspbian Buster release or later:
 
 ```shell
 sudo apt-get install build-essential qt5-qmake qtdeclarative5-dev qt5-default qttools5-dev-tools libqt5concurrent5 libjack-jackd2-dev 
```

Comment:
I think the libqt5concurrent5 is not needed since it's already in one of the packages mentioned before.

What was the difference between debian 11 and 10 (especially why is qt5-qmake now required)?

Reply:
OK yes remove libqt5concurrent5  then.  qt5-qmake is required for Ubuntu, so I assume it's required for Debian.

Reply:
Ok. I don't know about that. But I trust you.